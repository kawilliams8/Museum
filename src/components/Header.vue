<template>
  <div id="header">
    <h1>Museum</h1>
  </div>
</template>

<script>

export default {
  name: 'Header',
  components: {}
}
</script>

<style>
  #header {
    background-color: #F0F0F0;
    height: 100px;
  }
</style>