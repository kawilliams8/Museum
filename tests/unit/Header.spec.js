import { shallowMount } from "@vue/test-utils";
import Header from "@/components/Header.vue";

describe("Header.vue", () => {

  it('should pass the snapshot with props passed in', () => {
    const wrapper = shallowMount(Header);
    expect(wrapper.element).toMatchSnapshot();
  })
});