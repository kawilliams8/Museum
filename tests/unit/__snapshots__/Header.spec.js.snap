// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue should pass the snapshot with props passed in 1`] = `
<div
  id="header"
>
  <h1>
    Museum
  </h1>
</div>
`;
